reference/artifacts/from-build-triggers.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: single
+title:  "Artifacts from Build Triggers"
+sidebar:
+  nav: reference
+---
+
+## Overview
+
+When an external CI system triggers a pipeline, Spinnaker can use the CI build information to
+inject relevant artifacts into the pipeline. The instructions here assume that you have
+[set up a CI system](/setup/ci/) and are familiar with using
+[property files](/guides/user/pipeline/expressions/#property-files) to pass variables from
+CI builds to Spinnaker pipelines.
+
+The extraction of artifacts from the build information is done via a Jinja template; the
+template uses the trigger as context and outputs a list of artifacts to inject into the
+pipeline. Spinnaker provides a set of standard templates to use for artifact extraction,
```

Comment:
Link from "Spinnaker provides a set of standard templates" to new section at the end of the doc:

## Supplied templates